src/routes/+layout.svelte

```diff
@@ -3,8 +3,13 @@
 </script>
 
 <svelte:head>
-	<title>8 Dynamics | All We Can Save</title>
-	<meta name="description" content="Discover your 8 dynamics of climate change" />
+	<title>The All We Can Save 8 Dynamics Quiz</title>
+	<meta name="description" content="Explore your dynamics of climate engagement" />
+	<!-- Image tag must be FQDN -->
+	<meta
+		property="og:image"
+		content="https://dynamics-quiz.allwecansave.earth/images/dynamics_static.png"
```

Comment:
`BASE_URL` is a private env and can't be injected to this template file, and I was having issues finding a workaround that is dynamic but safe, considering our timeline to finish. This should do fine for now, until someone wants to update it

Reply:
sweet! I am good with this